app/templates/1_jobs.pug

```diff
@@ -1,4 +1,4 @@
-section#firstPage.special.section(data-scrollax-parent='true')
+section#firstPage.special.section(data-scrollax-parent='true', style='display:none')
```

Comment:
Maybe not that important, but wouldn't it be nicer to use a classname for hiding the sections? And add the css where it belongs instead of inline?